mmcls/models/backbones/efficientnet_v2.py

```diff
@@ -0,0 +1,330 @@
+        self.drop_path = DropPath(drop_path_rate) if drop_path_rate else nn.Identity()
+
+    def forward(self, x):
+        shortcut = x
+        x = self.conv(x)
+        if self.has_skip:
+            x = self.drop_path(x) + shortcut
+        return x
+
+
+@MODELS.register_module()
+class EfficientNetV2(BaseBackbone):
+    # repeat, kernel, stride, expansion, in_c, out_c, se_ratio, operator
+    # b0 is same as base
+    arch_settings = {
+        'base': [[1, 3, 1, 1, 32, 16, 0, 0], [2, 3, 2, 4, 16, 32, 0, 0],
+                 [2, 3, 2, 4, 32, 48, 0, 0], [3, 3, 2, 4, 48, 96, 0.25, 1],
+                 [5, 3, 1, 6, 96, 112, 0.25, 1],
+                 [8, 3, 2, 6, 112, 192, 0.25, 1]],
+        's': [[2, 3, 1, 1, 24, 24, 0, 0], [4, 3, 2, 4, 24, 48, 0, 0],
+              [4, 3, 2, 4, 48, 64, 0, 0], [6, 3, 2, 4, 64, 128, 0.25, 1],
+              [9, 3, 1, 6, 128, 160, 0.25, 1],
+              [15, 3, 2, 6, 160, 256, 0.25, 1]],
+        'm': [[3, 3, 1, 1, 24, 24, 0, 0], [5, 3, 2, 4, 24, 48, 0, 0],
+              [5, 3, 2, 4, 48, 80, 0, 0], [7, 3, 2, 4, 80, 160, 0.25, 1],
+              [14, 3, 1, 6, 160, 176, 0.25, 1],
+              [18, 3, 2, 6, 176, 304, 0.25, 1],
+              [5, 3, 1, 6, 304, 512, 0.25, 1]],
+        'l': [[4, 3, 1, 1, 32, 32, 0, 0], [7, 3, 2, 4, 32, 64, 0, 0],
+              [7, 3, 2, 4, 64, 96, 0, 0], [10, 3, 2, 4, 96, 192, 0.25, 1],
+              [19, 3, 1, 6, 192, 224, 0.25, 1],
+              [25, 3, 2, 6, 224, 384, 0.25, 1],
+              [7, 3, 1, 6, 384, 640, 0.25, 1]],
+        'xl': [[4, 3, 1, 1, 32, 32, 0, 0], [8, 3, 2, 4, 32, 64, 0, 0],
+               [8, 3, 2, 4, 64, 96, 0, 0], [16, 3, 2, 4, 96, 192, 0.25, 1],
+               [24, 3, 1, 6, 192, 256, 0.25, 1],
+               [32, 3, 2, 6, 256, 512, 0.25, 1],
+               [8, 3, 1, 6, 512, 640, 0.25, 1]],
+        'b0': [[1, 3, 1, 1, 32, 16, 0, 0], [2, 3, 2, 4, 16, 32, 0, 0],
+               [2, 3, 2, 4, 32, 48, 0, 0], [3, 3, 2, 4, 48, 96, 0.25, 1],
+               [5, 3, 1, 6, 96, 112, 0.25, 1],
+               [8, 3, 2, 6, 112, 192, 0.25, 1]],
+        'b1': [[2, 3, 1, 1, 32, 16, 0, 0], [3, 3, 2, 4, 16, 32, 0, 0],
+               [3, 3, 2, 4, 32, 48, 0, 0], [4, 3, 2, 4, 48, 96, 0.25, 1],
+               [6, 3, 1, 6, 96, 112, 0.25, 1],
+               [9, 3, 2, 6, 112, 192, 0.25, 1]],
+        'b2': [[2, 3, 1, 1, 32, 16, 0, 0], [3, 3, 2, 4, 16, 32, 0, 0],
+               [3, 3, 2, 4, 32, 56, 0, 0], [4, 3, 2, 4, 56, 104, 0.25, 1],
+               [6, 3, 1, 6, 104, 120, 0.25, 1],
+               [10, 3, 2, 6, 120, 208, 0.25, 1]],
+        'b3': [[2, 3, 1, 1, 40, 16, 0, 0], [3, 3, 2, 4, 16, 40, 0, 0],
+               [3, 3, 2, 4, 40, 56, 0, 0], [5, 3, 2, 4, 56, 112, 0.25, 1],
+               [7, 3, 1, 6, 112, 136, 0.25, 1],
+               [12, 3, 2, 6, 136, 232, 0.25, 1]]
+    }
+
+    def __init__(self,
+                 arch: str = 's',
+                 drop_path_rate: float = 0.,
+                 out_channels: int = 1280,
```

Comment:
`out_channels` should not be here;

Reply:
If such a parameter is necessary, I think it would be better to put it in arch, which is a variable of a particular ''arch''.